site/content/about/_index.md

```diff
@@ -26,39 +26,48 @@ We create partnerships with other organizations and communities; both those that
 
 ## Team
 
-Techqueria is possible thanks to the work of lots of people, organizers, volunteers, mentors, sponsors, and advisors; and we're thankful for all of them.
+Techqueria is an all-volunteer organiztion that is made possible thanks to the work of lots of organizers, mentors, sponsors, and advisors. Thank you all!
 
 ### Co-Founders
 
-- [David Silva](https://linkedin.com/in/dvidsilva), co-founder of Techqueria & senior developer at Slingshot Health
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
+- [David Silva](https://linkedin.com/in/dvidsilva), Co-founder of Techqueria & Senior Developer at Slingshot Health
+- [Shashi Jain](https://www.linkedin.com/in/quahada), Co-founder of Techqueria & Software Engineer
 
 ### Organizers
 
 #### Bay Area
-
-- [Shashi Jain](https://www.linkedin.com/in/quahada), co-founder of Techqueria & software engineer at Diamond Web Services
-- [David Hincapie](https://www.linkedin.com/in/davidohincapie/), product specialist at Oculus VR
-- [Philip Ventura](https://www.linkedin.com/in/fvntr/), software engineer at Lending Club
+- [Rubi Sanchez](https://www.linkedin.com/in/rubisanchez), Product Manager at AdRoll
```

Comment:
There should be a space below any headers so it's consistent with rest of headers.